accessibility-checker.php

```diff
@@ -253,11 +253,7 @@ function edac_update_database() {
 	// Update database version option.
 	$option_name = 'edac_db_version';
 	$new_value   = EDAC_DB_VERSION;
-	if ( get_option( $option_name ) !== false ) {
-		update_option( $option_name, $new_value );
-	} else {
-		add_option( $option_name, $new_value );
-	}
+	update_option( $option_name, $new_value );
```

Comment:
There's no reason to have this condition here. `update_option` takes care of both scenarios for us.